src/plugins/mailer/index.ts

```diff
@@ -20,9 +20,14 @@ export interface MailerOptions {
   fromEmail: string;
 }
 
+export interface Item {
```

Comment:
We already have an entity called `Item`. I think we could use something more descriptive:
```suggestion
export interface ListItem {
```